src/app.jsx

```diff
@@ -459,6 +464,170 @@ class Application extends React.Component {
             this.onOwnerChanged("all");
     }
 
+    // Read information about quadlets from /run/ until podman provides a remote API for this.
+    // https://github.com/containers/podman/issues/27119
+    async initQuadlets(con) {
+        let path = "/run/systemd/generator";
+        let quadlets = { pods: {}, containers: {} };
+
+        if (con.uid === null) {
+            path = sessionStorage.getItem('XDG_RUNTIME_DIR') + '/systemd/generator';
+        } else if (con.uid !== 0) {
+            console.warn(`unsupported connection ${con.uid} for loading quadlets`);
+            this.setState(prevState => {
+                const users = prevState.users.map(u => u.uid === con.uid ? { ...u, quadletsLoaded: true } : u);
+                return { users };
+            });
+            return;
+        }
+
+        try {
+            const quadlets_str = await python.spawn(scan_quadlets, [path]);
+            quadlets = JSON.parse(quadlets_str);
+        } catch (exc) {
+            console.warn("error during discovering of quadlets", exc);
+            this.setState(prevState => {
+                const users = prevState.users.map(u => u.uid === con.uid ? { ...u, quadletsLoaded: true } : u);
+                return { users };
+            });
+            return;
+        }
+
+        // { service_name-uid: { } }
+        this.setState(prevState => {
+            const podNameServiceMap = {};
+
+            const copyQuadletPods = {};
+            // keep/copy the pods of other users
+            Object.entries(prevState.quadletPods || {}).forEach(([id, container]) => {
+                if (container.uid !== con.uid)
+                    copyQuadletPods[id] = container;
+            });
+
+            for (const key of Object.keys(quadlets.pods)) {
+                const quadlet_pod = quadlets.pods[key];
+                const container_key = makeKey(con.uid, key);
+
+                const pod = {
+                    uid: con.uid,
+                    key: container_key,
+                    Id: container_key,
+                    Status: "Exited",
+                    Name: quadlet_pod.name,
+                    Labels: {
+                        PODMAN_SYSTEMD_UNIT: key,
+                    }
+                };
+                copyQuadletPods[pod.key] = pod;
+
+                // The key is the service name, but that isn't used in reference
+                podNameServiceMap[basename(quadlet_pod.source_path)] = key;
+            }
+
+            const copyQuadletContainers = {};
+            // keep/copy the containers of other users
+            Object.entries(prevState.quadletContainers || {}).forEach(([id, container]) => {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2266-fa7f0a3b-20250922-124438-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)